### PR TITLE
Feat/post content style

### PR DESCRIPTION
**Pull request by KaiElwood**
Added all required post styles. #62

### CODE REVIEW

File: wp-content/themes/csisnuclearnetwork/assets/_scss/pages/single.scss [hunk 1]
```diff
@@ -81,48 +88,53 @@
     a[href*='//']:not([class])
     {
       /* stylelint-enable */
-      // color: $color-link-primary-200;
-      // font-family: $font__merriweather;
-      @include font-size(18px);
+      color: $color-accent-primary-blue-200;
+      // @include font-size(18px);
```
Comment (urchykli) [inline wp-content/themes/csisnuclearnetwork/assets/_scss/pages/single.scss:92]: Let's remove this unused code.

File: wp-content/themes/csisnuclearnetwork/assets/_scss/pages/single.scss [hunk 2]
```diff
@@ -81,48 +88,53 @@
...
       font-weight: 400;
```
Comment (urchykli) [inline wp-content/themes/csisnuclearnetwork/assets/_scss/pages/single.scss:93]: We can remove this.

File: wp-content/themes/csisnuclearnetwork/assets/_scss/pages/single.scss [hunk 3]
```diff
@@ -81,48 +88,53 @@
...
+    //     background-image: url('/wp-content/themes/nuclearnetwork/assets/static/icons/arrow-external.svg');
+    //     background-size: 100% 100%;
+    //     filter: invert(38%) sepia(89%) saturate(439%) hue-rotate(133deg) brightness(86%) contrast(85%);
+    //     transition: filter 0.3s ease-in-out;
+    //   }
+
+    //   &:hover::after {
+    //     filter: invert(33%) sepia(14%) saturate(1559%) hue-rotate(131deg) brightness(95%) contrast(87%);
+    //   }
+    // }
```
Comment (urchykli) [inline wp-content/themes/csisnuclearnetwork/assets/_scss/pages/single.scss:137]: Did Christina say we aren't using the icon for external links?

**Review by urchykli — CHANGES_REQUESTED** [review]
Nice job on this, Kai!  Just a few comments and questions including the ones inline.  Thanks!

- On line 53, let's remove `ul`, since the correct color is added on line 69.
- If you aren't using the commented out code, please delete it.

File: wp-content/themes/csisnuclearnetwork/assets/_scss/pages/single.scss [hunk 4]
```diff
@@ -113,28 +108,28 @@
...
+        width: rem(14);
+        height: rem(14);
+        margin-right: 5px;
+        background-image: url('/wp-content/themes/csisnuclearnetwork/assets/static/icons/arrow-external.svg');
+        background-size: 100% 100%;
+        filter: invert(22%) sepia(87%) saturate(1176%) hue-rotate(186deg) brightness(101%) contrast(85%);
+        transition: filter 0.3s ease-in-out;
+      }
+
+      &:hover::after {
```
Comment (urchykli) [inline wp-content/themes/csisnuclearnetwork/assets/_scss/pages/single.scss:129]: We need a focus state as well.

**Review by urchykli — CHANGES_REQUESTED** [review]
Nice work, Kai!  We still need to remove the `ul` from line 53 and make the small inline changes, but then we're ready to merge.

File: wp-content/themes/csisnuclearnetwork/assets/_scss/pages/single.scss [hunk 5]
```diff
@@ -89,8 +87,6 @@
     {
       /* stylelint-enable */
       color: $color-accent-primary-blue-200;
-      // @include font-size(18px);
-      font-weight: 400;
       text-decoration: underline;
 
       &:visited {
```
[the remark this answers is not in the document]
Comment (KaiElwood) [reply wp-content/themes/csisnuclearnetwork/assets/_scss/pages/single.scss:92]: Got it

**Review by urchykli — APPROVED** [review]

**Pull request merged**